cypress/tests/check-tabs.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { K8S_KIND } from '../utils/const/index';
+import * as vmView from '../views/selector';
+import { tab } from '../views/tab';
+import { vm } from '../views/vm';
+
+describe('Check all virtualization pages can be loaded', () => {
+  before(() => {
+    cy.login();
+    cy.deleteResource(K8S_KIND.VM, 'vm-example', 'default');
+    cy.deleteResource(K8S_KIND.TEMPLATE, 'vm-template-example', 'default');
+  });
+
+  after(() => {
+    cy.deleteResource(K8S_KIND.VM, 'vm-example', 'default');
+    cy.deleteResource(K8S_KIND.TEMPLATE, 'vm-template-example', 'default');
+  });
+
+  describe('Check VM and VMI list and tabs', () => {
+    it('create vm-example', () => {
+      cy.visitVMs();
+      vm.createVMFromYAML();
+      cy.contains('Hostname').should('be.visible');
+
+      // start the vm
+      cy.byButtonText('Actions').click();
+      cy.get(vmView.vmActionStart).click();
+      cy.contains(vmView.vmStatusOnOverview, 'Starting').should('exist');
+      // VM in CI cannot become running
+      // cy.contains(vmView.vmStatusOnOverview, 'Running', { timeout: 600000 }).should('exist');
```

Comment:
is this left intentionally?

Reply:
yes, it seems the VM cannot get into running in the CI.
I added a comment in line 28.